cluster/charts/rook-ceph/values.yaml

```diff
@@ -313,3 +313,12 @@ discoverDaemonUdev:
 
 # Whether the OBC provisioner should watch on the operator namespace or not, if not the namespace of the cluster will be used
 enableOBCWatchOperatorNamespace: true
+
+admissionController:
+  # Set admissionControllerTolerations and admissionControllerNodeAffinity for admission controller pod.
+  # The admission controller would be best to start on the same nodes as other ceph daemons.
+  # admissionControllerTolerations:
```

Comment:
You have used the variable as just `tolerations` above in the template, same for the nodeAffinity on line 324.